### DISCUSSION TITLE
Welcome to postgresml Discussions!

### DISCUSSION THREAD

**Discussion by montanalow** [postgresml/postgresml#170] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by abevieiramota** [comment]
Hi people, I'm Abelardo, an IT analyst based in Brazil and I'm glad to find a tool that mixes a SGBD that I love, Postgres, with an area that I'm studying, Machine Learning! Thank you all for your effort developing it! 

I got here searching for any resource on how to contribute with this project, as I'm interested in learning how to develop Postgres extensions and how PostgresML works.

**Reply by abevieiramota** [reply]
Ok, just found out the Developer Guide page!
